tests/core/conftest.py

```diff
@@ -11,6 +11,9 @@
     EventBus,
 )
 
+EndpointPair = Tuple[Endpoint, Endpoint]
+EndpointTriplet = Tuple[Endpoint, Endpoint, Endpoint]
```

Comment:
There is! We run mypy on the tests because it allows us to test if we get the type inference guarantees that we expect to get. E.g. when we make a request, we want to be sure that the response isn't seen as `Any` which would mean that any property access would be allowed even if the property does not exist.

We run `mypy` in `strict` mode in this project and running it on the tests would mean that e.g. the following line would throw if for some reason we have a regression with the type inference that we are assuming.

https://github.com/ethereum/lahja/blob/6a1a45439c9eaac301a08991cec60d54940767be/tests/core/test_basics.py#L41-L43